### PR TITLE
grab dragDropManager from context

### PR DESCRIPTION
**Pull request by Morantron**
Fixes the infamous @andresgutgon bug https://github.com/gaearon/react-dnd/issues/186 :trollface:

### CODE REVIEW

File: src/Kanban/index.js [hunk 1]
```diff
@@ -14,6 +13,16 @@ import SortableList from '../SortableList';
 
 const GridWithScrollZone = withScrolling(Grid);
 const horizontalStrength = createHorizontalStrength(200);
+import { DragDropManager } from 'dnd-core';
```
Comment (andresgutgon) [inline src/Kanban/index.js]: Maybe It would be good idea require this as dependency

Comment (Morantron) [inline src/Kanban/index.js]: oopsie

**Comment by andresgutgon** [comment]
📯 👍

**Comment by edulan** [comment]
:clap: :clap: 

Thanks @andresgutgon :smile:

**Pull request merged**